api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -191,6 +206,23 @@ def from_orm(cls, offer: Offer) -> "OfferResponse":
 
         if result.extraData:
             result.extraData.durationMinutes = offer.durationMinutes
+
+            if offer.extraData:
+                gtl_id = offer.extraData.get("gtl_id")
+                if gtl_id is not None and gtl_id in GTLS:
+                    gtl_infos = GTLS[gtl_id]
+                    label = gtl_infos.get("label")
+                    if label:
+                        result.extraData.gtls = gtls(
+                            label=label,
+                            level01Label=gtl_infos.get("level_01_label"),
+                            level02Label=gtl_infos.get("level_02_label"),
+                            level03Label=gtl_infos.get("level_03_label"),
+                            level04Label=gtl_infos.get("level_04_label"),
+                        )
+                    else:
+                        logger.error("GTL label not found for id %s", gtl_id)
+                        result.extraData.gtls = None
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏C'est cool, c'est défensif 
Je propose de mettre ça dans une fonction `get_gtl_labels(offer: Offer) -> GtlLabels | None` pour alléger la lecture ici